openhtf/output/proto/mfg_event_converter.py

```diff
@@ -23,6 +23,7 @@
 from openhtf.util import data as htf_data
 from openhtf.util import units
 from openhtf.util import validators
+from openhtf.util.json_encoder import TestRecordEncoder
```

Comment:
Style: Do not import classes (or functions) from libraries.  Use the library namespace.

from openhtf.util import json_encoder